pkgs/top-level/aliases.nix

```diff
@@ -1185,6 +1185,9 @@ mapAliases {
   inotifyTools = inotify-tools;
   insync-emblem-icons = throw "'insync-emblem-icons' has been removed, use 'insync-nautilus' instead"; # Added 2025-05-14
   inter-ui = throw "'inter-ui' has been renamed to/replaced by 'inter'"; # Converted to throw 2024-10-17
+  intel-compute-runtime = throw "intel-graphics-compiler has been removed, as it does not build with supported LLVM versions; consider using Mesa’s Rusticl implementation of OpenCL"; # Added 2025-08-22
```

Comment:
I've spent way to much time already trying to support the GPU and CPU hardware acceleration features of my devices and from that I learned that things which should be compatible in theory are not always compatible in practice and there exist some programs which only work with one variant of hardware acceleration and not another one. I have a really hard time believing that this switch generally just works for all programs and think this is not a great migration path for all things.

Reply:
FWIW, my understanding is that Rusticl explicitly targets being able to run SYCL programs, but I don’t know the exact level of compatibility or performance there.